packages/jest-config/src/ValidConfig.js

```diff
@@ -100,7 +101,10 @@ export default ({
   testMatch: ['**/__tests__/**/*.js?(x)', '**/?(*.)+(spec|test).js?(x)'],
   testNamePattern: 'test signature',
   testPathIgnorePatterns: [NODE_MODULES_REGEXP],
-  testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.jsx?$',
+  testRegex: MultipleValidOptions(
```

Comment:
also, what do you think about naming this just `multiple` for simplicity? I'm good with both, so pick whatever you like :)

Reply:
I don't feel strongly, but I guess I'd lean towards leaving it more verbose to provide clear context, since `Mutliple` could potentially mean more than one thing.